src/Illuminate/Validation/Rules/File.php

```diff
@@ -330,9 +330,9 @@ protected function buildMimetypes()
      */
     protected function fail($messages)
     {
-        $messages = (new Collection(Arr::wrap($messages)))->map(function ($message) {
-            return $this->validator->getTranslator()->get($message);
-        })->all();
+        $messages = (new Collection(Arr::wrap($messages)))
+            ->map(fn ($message) => $this->validator->getTranslator()->get($message))
```

Comment:
Not sure about this one actually, as `$this->validator->getTranslator()` would be called immediately in my suggestion, but I just throw it in 🤔 
```suggestion
            ->map([ $this->validator->getTranslator(), 'get')
```
```suggestion
            ->map($this->validator->getTranslator()->get(...))
```